ecl/tests/unit/network/v2/test_wasabi_gateways.py

```diff
@@ -0,0 +1,49 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+import testtools
+
+from ecl.network.v2 import wasabi_gateways
+
+IDENTIFIER = 'IDENTIFIER'
+EXAMPLE = {
+    "id": IDENTIFIER,
+    "common_function_gateway": IDENTIFIER,
+    "description": "Example wasabi gateway 1 description.",
+    "name": "Example wasabi gateway 1",
+    "status": "ACTIVE",
+    "tenant_id": IDENTIFIER
+}
+
+
+class TestWasabiGateways(testtools.TestCase):
+
+    def test_basic(self):
+        sot = wasabi_gateways.WasabiGateways()
+        self.assertEqual('wasabi_gateway', sot.resource_key)
+        self.assertEqual('wasabi_gateways', sot.resources_key)
+        self.assertEqual('/wasabi_gateways', sot.base_path)
+        self.assertEqual('wasabi_gateway', sot.service.service_type)
```

Comment:
```suggestion
        self.assertEqual('network', sot.service.service_type)
```
serviceに設定しているのは NetworkServiceですから、ここは "network"という値になるかと思います。